src/test/java/uk/gov/digital/ho/proving/income/validator/CatBNonSalariedIncomeValidatorTest.java

```diff
@@ -0,0 +1,274 @@
+package uk.gov.digital.ho.proving.income.validator;
+
+import org.junit.Test;
+import uk.gov.digital.ho.proving.income.api.SalariedThresholdCalculator;
+import uk.gov.digital.ho.proving.income.api.domain.CheckedIndividual;
+import uk.gov.digital.ho.proving.income.validator.domain.ApplicantIncome;
+import uk.gov.digital.ho.proving.income.validator.domain.IncomeValidationRequest;
+import uk.gov.digital.ho.proving.income.validator.domain.IncomeValidationResult;
+import uk.gov.digital.ho.proving.income.validator.domain.IncomeValidationStatus;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.digital.ho.proving.income.validator.CatASalariedTestData.getDate;
+import static uk.gov.digital.ho.proving.income.validator.CatBNonSalariedTestData.*;
+
+public class CatBNonSalariedIncomeValidatorTest {
```

Comment:
This has good coverage around various different scenarios, we could do with one big nasty test that covers off several scenarios to make sure that they work in tandem.  Just the one though...